funnel/static/js/libs/mui.js

```diff
@@ -0,0 +1,1099 @@
+// "version": "0.9.21"
```

Comment:
This is not minimized. Prettier will automatically unminimize this as part of pre-commit, so it has to be added in exclusions. Ideally this comes from webpack.